doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,317 @@
+.. _ncs_release_notes_100:
+
+|NCS| v1.0.0 Release Notes
+##########################
+
+nRF Connect SDK delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products. It includes the MCUboot and the Zephyr RTOS open source projects which are continuously integrated and re-distributed with the SDK.
+
+nRF Connect SDK v1.0.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+* |NCS| v1.0.0 is aligned with the production release of nRF9160.
+* Added a new AWS FOTA sample demonstrating application firmware update through the use of AWS IoT Jobs with MQTT and HTTP.
+* Added secure services in Secure Partition Manager (SPM) for TrustZone devices (nRF9160).
+* Updated the Download Client library with support for HTTPS and IPv6.
+* Included a new version of the nRF BLE Controller.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.0.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
```

Comment:
Would like to add the following section (similar to what we have for SD supported in nRF5 SDK):

Supported Modem FW:
- mfw_nrf9160_1.0.0

Reply:
done